Extension/src/LanguageServer/lmTool.ts

```diff
@@ -235,7 +235,10 @@ export class CppConfigurationLanguageModelTool implements vscode.LanguageModelTo
                 contextString += `The project targets the ${chatContext.targetArchitecture} architecture. `;
                 telemetryProperties["targetArchitecture"] = chatContext.targetArchitecture;
             }
-
+            if (chatContext.usedTestFrameworks?.length > 0) {
+                contextString += `The active document contains the content of the file '${currentDoc.fileName}' which is located in a directory structure where other files are including headers for the following C++ test frameworks: ${chatContext.usedTestFrameworks.join(', ')} . `;

```

Comment:
There is an unnecessary space before and after the `.`, i.e. ` . ` -> `.`

Reply:
@sean-mcmanus the space after the dot it is intentional as the constructions is elsewhere. The one before, either way works for me, I'll probably remove it.